js/thankyou.js

```diff
@@ -27,8 +27,11 @@
 'Go raibh maith agat as do chuid ranníocaíochtaí!',
 'Terimakasih atas kontribusi anda!',
 'Köszönjük a hozzájárulást!',
+'Takk fyrir þitt framlag!',
```

Comment:
Appearing two times. Please delete this line

Reply:
Okay